src/pages/en/core-concepts/astro-pages.md

```diff
@@ -5,17 +5,30 @@ description: An introduction to Astro pages
 i18nReady: true
 ---
 
-**Pages** are a special type of [Astro component](/en/core-concepts/astro-components/) that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
+**Pages** are special files that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
 
 ### File-based routing
 
-Astro leverages a routing strategy called **file-based routing.** Every `.astro` file in your `src/pages` directory becomes a page or an endpoint on your site based on its file path.
+Astro leverages a routing strategy called **file-based routing.**. Each page file in your `src/pages` directory becomes an endpoint on your site based on its file path.
```

Comment:
Nit: I think “file” is enough here maybe? Also do we have the convention of using a trailing slash on directories like in the opening paragraph?

```suggestion
Astro leverages a routing strategy called **file-based routing.**. Each file in your `src/pages/` directory becomes an endpoint on your site based on its file path.
```

Reply:
I don’t think Astro likes when you put non-page files in `src/pages/` IIRC? So in practice _unless_ using the `_` prefix trick, you should have _only_ pages in `src/pages/`?

I guess similar to “special”, I’m not sure sure how I make my file a _page_ file? So wanted to focus on location rather than file type.

Reply:
Chaos monkey! What if we swap positions of supported page file types and file-based routing, so that when we get to this line in routing, people already have "supported file types" in their head? Does this then become easier to write/reference?

Reply:
Swapping! 🙊